dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -56,6 +56,10 @@ namespaces:
         type: timer
         tags: [ channel-name ]
         docs: Time spent waiting in the sticky queue before execution attempt.
+      requests.size:
+        type: histogram
+        tags: [channel-name, service-name, endpoint, retryable]
```

Comment:
Thoughts on updating from `retryable` to `repeatable` matching the RequestBody API?

Defer to your judgement, just mentioning it in case you aren't familiar with teh feature: Metric-schema allows us to enumerate tag values which may improve the generated documentation slightly (and avoids unnecessary string values in code):
```yaml
tags:
  - name: repeatable
     values: ['true', 'false']
```